main.py

```diff
@@ -190,7 +213,7 @@ def command_skip():
     active_voice_client.stop()
 
 
-def play_next_song(e=None):
+def play_next_song(e=None, skip_count=0):
```

Comment:
```suggestion
def play_next_song(e: BaseException = None, skip_count: int = 0):
```

Hah, I also forgot to add types. Might as well fix while we're here.